Lib/test/test_pprint.py

```diff
@@ -346,6 +346,36 @@ def test_mapping_proxy(self):
                           ('lazy', 7),
                           ('dog', 8)]))""")
 
+    def test_empty_simple_namespace(self):
+        self.assertEqual(pprint.pformat(types.SimpleNamespace()), "namespace()")
+
+    def test_small_simple_namespace(self):
+        n = types.SimpleNamespace(a=1, b=2)
+        self.assertEqual(pprint.pformat(n), "namespace(a=1, b=2)")
```

Comment:
Likewise with one action per line.

```suggestion
        self.assertEqual(
                pprint.pformat(ns),
                "namespace(a=1, b=2)")
```